drivers/i2c/i2c_ll_stm32.c

```diff
@@ -300,6 +305,10 @@ static const struct i2c_stm32_config i2c_stm32_cfg_##name = {		\
 	},								\
 	STM32_I2C_IRQ_HANDLER_FUNCTION(name)				\
 	.bitrate = DT_PROP(DT_NODELABEL(name), clock_frequency),	\
+	COND_CODE_1(CONFIG_I2C_STM32_V2,				\
```

Comment:
If possible, cf previous comment:
```suggestion
	COND_CODE_1(DT_HAS_COMPAT_STATUS_OKAY(i2c_ll_stm32_v2),				\
```